docs/ansible/roles/debops.docker/defaults-detailed.rst

```diff
@@ -0,0 +1,196 @@
+Default variable details
+========================
+
+.. include:: ../../../includes/global.rst
+
+Some of ``debops.docker`` default variables have more extensive configuration
+than simple strings or lists. Here you can find documentation and examples for
+them.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _docker_ref_networks:
+
+docker__*_networks
+------------------
+
+The ``docker__*_networks`` variables define the Docker networks which should be
+managed by Ansible. These variables are lists of networks with parameters from
+the `Ansible docker_network module`_.
+
+Here are some of the more important parameters:
+
+``item.name``
+  Required. The name of the network.
+
+``item.connected``
+  Optional. List of container names or IDs to connect to the network.
+
+``item.appends``
+  Optional. Set to ``True`` to prevent the removal of a container from the
+  network when the container name or ID is removed from the ``item.connected``
+  list.
+
+``item.state``
+  Optional. If not specified or ``present``, the network will be configured on
+  the Docker host. If ``absent``, the network will be removed.
+
+Examples
+~~~~~~~~
```

Comment:
The "Examples" section should be put before the "Syntax" section, that way you get a quick overview of how the configuration is written, and you can look up the specific syntax later on.